examples/TurnOnFujitsuAC/platformio.ini

```diff
@@ -13,6 +13,10 @@ build_flags = ; -D_IR_LOCALE_=en-AU
 platform = espressif8266
 board = nodemcuv2
 
+[env:d1_mini]
+platform = espressif8266
+board = d1_mini
+
```

Comment:
Do we need this extra env? (i.e. How long extra does it add to the travis build time?)

Reply:
I added it when testing and it seems no need to add it in release.

Reply:
That's what I figured may have been the case. Please remove.